verify/src/metadata_schema.json

```diff
@@ -123,6 +123,28 @@
         },
         "perpendicular_baseline": {
           "type": "number"
+        },
+        "version": {
```

Comment:
reviewing past .json files in our archive, they appear to all have `version`. We can add `version` to `required`, above.